packages/fuels/tests/Forc.toml

```diff
@@ -0,0 +1,71 @@
+[workspace]
```

Comment:
I think we should have a check to ensure all tests directories are listed here. Maybe open an issue for this?

Reply:
Maybe. If we don't decide that we will leave it as it was, because we don't use force as a dependency